.flake8

```diff
@@ -3,9 +3,12 @@ max-complexity = 6
 inline-quotes = double
 max-line-length = 88
 extend-ignore = E203
-docstring_style=sphinx
```

Comment:
I would expect `docstring-convention=google`, as described at https://github.com/pycqa/flake8-docstrings .

Did you remove it because by default includes Google?